### ISSUE TITLE
Cannot import name 'ModelMetaclass' from 'pydantic.main'

### ISSUE THREAD

**Issue by ztobs** [stitionai/devika#335]
**Describe the bug**
When I run the `python3 devika.py` command, I am faced with `ImportError: cannot import name 'ModelMetaclass' from 'pydantic.main'`error, who know how I can fix this?

Below is the full traceback

```
File "/home/donztobs/Workspace/Projects/devika/devika.py", line 10, in <module>
    init_devika()
  File "/home/donztobs/Workspace/Projects/devika/src/init.py", line 24, in init_devika
    from src.bert.sentence import SentenceBert
  File "/home/donztobs/Workspace/Projects/devika/src/bert/sentence.py", line 1, in <module>
    from keybert import KeyBERT
  File "/home/donztobs/.local/lib/python3.10/site-packages/keybert/__init__.py", line 2, in <module>
    from keybert._model import KeyBERT
  File "/home/donztobs/.local/lib/python3.10/site-packages/keybert/_model.py", line 17, in <module>
    from keybert.llm._base import BaseLLM
  File "/home/donztobs/.local/lib/python3.10/site-packages/keybert/llm/__init__.py", line 28, in <module>
    from keybert.llm._langchain import LangChain
  File "/home/donztobs/.local/lib/python3.10/site-packages/keybert/llm/_langchain.py", line 3, in <module>
    from langchain.docstore.document import Document
  File "/home/donztobs/.local/lib/python3.10/site-packages/langchain/__init__.py", line 6, in <module>
    from langchain.agents import MRKLChain, ReActChain, SelfAskWithSearchChain
  File "/home/donztobs/.local/lib/python3.10/site-packages/langchain/agents/__init__.py", line 2, in <module>
    from langchain.agents.agent import (
  File "/home/donztobs/.local/lib/python3.10/site-packages/langchain/agents/agent.py", line 15, in <module>
    from langchain.agents.tools import InvalidTool
  File "/home/donztobs/.local/lib/python3.10/site-packages/langchain/agents/tools.py", line 8, in <module>
    from langchain.tools.base import (
  File "/home/donztobs/.local/lib/python3.10/site-packages/langchain/tools/__init__.py", line 3, in <module>
    from langchain.tools.base import BaseTool
  File "/home/donztobs/.local/lib/python3.10/site-packages/langchain/tools/base.py", line 16, in <module>
    from pydantic.main import ModelMetaclass
ImportError: cannot import name 'ModelMetaclass' from 'pydantic.main' (/home/donztobs/.local/lib/python3.10/site-packages/pydantic/main.py)

```

**Comment by ztobs** [comment]
I solved the issue by upgrading my python3 from 3.10 to 3.11 and pip from 22 to 24.

This article guided me through https://medium.com/python-mvp/upgrade-python-3-10-to-3-11-on-ubuntu-22-04-2f63e4d326f9

**Comment by ztobs** [comment]
I solved the issue by upgrading my python3 from 3.10 to 3.11 and pip from 22 to 24.

This article guided me through https://medium.com/python-mvp/upgrade-python-3-10-to-3-11-on-ubuntu-22-04-2f63e4d326f9
